### PR TITLE
#5986 Adding .DS_Store to Scaffolded Backstage app .gitignore file

### PR DESCRIPTION
**Pull request by dortegau**
Signed-off-by: Daniel Ortega <dortega@idealista.com>

#### :heavy_check_mark: Checklist

I've just added `.DS_Store` pattern in Scaffolded Backstage App .gitignore file

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by Rugvip — COMMENTED** [review]
Thanks! Needs a changeset thought, including some short instructions on how to migrate an existing app. See https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md for examples of that

**Comment by dortegau** [comment]
Added! :)

File: .changeset/clean-frogs-brake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': minor
+---
+
+Adding .DS_Store pattern to .gitignore in Scaffolded Backstage App. To migrate an existing app that pattern should be added manually.
```
Comment (Rugvip) [inline .changeset/clean-frogs-brake.md:5]: I think a diff could be a helpful addition here, something like

```diff
+# macOS
+.DS_Store
```

File: .changeset/clean-frogs-brake.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': minor
```
Comment (Rugvip) [inline .changeset/clean-frogs-brake.md:2]: Let's make this `patch` since we consider changes to the templates to be non-breaking changes.

**Review by Rugvip — APPROVED** [review]
🎉

**Pull request merged**